coyote/views.py

```diff
@@ -24,6 +24,11 @@ def get_domain(url):
 
 
 def home(request):
+    """
+    it takes necessary data from frontend request and sent it to hackerearth API
```

Comment:
`it takes necessary data` => `it takes the inputted code`
See, for a newcomer, the explanation provided must be clear. The explanation seems to be written by someone who did not give enough time to read the function.

`sent it to hackerearth API` => `sends it to hackerearth API` (although technically incorrect, still acceptable)